packages/web/src/components/user-name-and-badges/UserNameAndBadges.tsx

```diff
@@ -33,14 +33,14 @@ type UserNameAndBadgesProps =
 
 const UserNameAndBadgesImpl = (props: UserNameAndBadgesImplProps) => {
   const { user, onNavigateAway, classes } = props
-  const goToRoute = useGoToRoute()
+  const navigate = useNavigateToPage()
```

Comment:
I'm curious what motivated getting rid of `goToRoute`?

Reply:
the fictions do the same thing. literally the same impl haha